test/testfixture/testfixture.go

```diff
@@ -155,6 +157,7 @@ const (
 	kindWorkItemLinkCategories kind = "work_item_link_categorie"
 	kindWorkItemLinks          kind = "work_item_link"
 	kindLabels                 kind = "labels"
+	kindTrackers               kind = "trackers"
```

Comment:
Can you please make this singular: `trackers`->`tracker`? Also please fix `labels` -> `label` and `work_item_link_categorie` -> `work_item_link_category` when you're doing that.

Reply:
@kwk should all kind names be singular or just labels and work_item_link_categories ?

Reply:
What do you mean? Do you see any more?

Reply:
oh sorry. I understand now what changes you want. @kwk

Reply:
@kwk fixed it here d768e79